pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.pkgproj

```diff
@@ -11,10 +11,13 @@
     <ProjectReference Include="..\Microsoft.NETCore.DotNetHost\Microsoft.NETCore.DotNetHostPolicy.pkgproj">
       <TargetFramework>.NETCoreApp1.0</TargetFramework>
     </ProjectReference>
+    <ProjectReference Include="..\Microsoft.NETCore.DotNetHost\Microsoft.NETCore.DotNetHostPolicy.pkgproj">
+      <TargetFramework>.NETCoreApp1.1</TargetFramework>
+    </ProjectReference>
   </ItemGroup>
-
+  
   <Target Name="ReadDependenciesFromProjectJsonFile" BeforeTargets="GetPackageDependencies">
-    <ReadLinesFromFile File="$(MSBuildThisFileDirectory)project.json">
+    <ReadLinesFromFile File="$(MSBuildThisFileDirectory)\netcoreapp1.0\project.json">
```

Comment:
I couldn't find an elegant way to do this in msbuild syntax. This code is duplicated below for netcoreapp1.1.

We want to import both project.jsons (for netcoreapp1.0 and netcoreapp1.1) so that the meta package Microsoft.NETCore.App contains <targetframework> tags for both.